OasisDEX/oasis-borrow
# Vault table adjustment

# [Vault table adjustment](https://app.shortcut.com/oazo-apps/story/8547/spike-1-day-understand-changes-required-in-db-support-ajna-borrow-and-multiply-positions)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added new `protocol` column
- changed constrain to include `protocol`
- updated all places in codebase where we were using logic related to create / update / read position type (new constrain requires passing all params included in constrain)
  
## How to test 🧪
  <Please explain how to test your changes>

- after you run migration visit borrow / multiply maker vaults to see whether they load properly
- try to change type from borrow to multiply and in opposite way using your own position
- verify whether protocol is stored properly in db
